traffic_portal/app/src/common/modules/form/deliveryServiceSslKeys/generate/form.GenerateDeliveryServiceSslKeys.tpl.html

```diff
@@ -30,7 +30,16 @@
     <div class="x_content">
         <br>
         <form name="dsGenerateSslKeyForm" class="form-horizontal form-label-left" novalidate>
-            <div class="form-group" ng-class="{'has-error': hasError(dsGenerateSslKeyForm.country), 'has-feedback': hasError(dsGenerateSslKeyForm.country)}">
+            <div class="form-group">
+                <label class="control-label col-md-2 col-sm-2 col-xs-12">Use Let's Encrypt</label>
```

Comment:
`label`s should have a `for` attribute indicating what it is they label.

Reply:
done